src/test/java/org/springframework/integration/aws/s3/config/xml/AmazonS3OutboundChannelAdapterParserAWSTests.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2002-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.integration.aws.s3.config.xml;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.integration.aws.s3.core.AmazonS3Operations;
+import org.springframework.integration.endpoint.EventDrivenConsumer;
+import org.springframework.integration.handler.advice.AbstractRequestHandlerAdvice;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHandler;
+import org.springframework.messaging.support.GenericMessage;
+
+import static org.junit.Assert.assertEquals;
+import static org.springframework.integration.test.util.TestUtils.getPropertyValue;
+
+/**
+ * The test case for the aws-s3 namespace's {@link AmazonS3OutboundChannelAdapterParser} class by
+ * connecting to AWS.
+ *
+ * @author Karthikeyan Palanivelu
+ *
+ * @since 1.0
+ *
+ */
+@Ignore
+public class AmazonS3OutboundChannelAdapterParserAWSTests {
+
+
+	/**
+	 * Test case for the xml definition with a custom implementation of {@link AmazonS3Operations}
+	 *
+	 */
+	private volatile static int adviceCalled;
+
+	/**
+	 * Tests the outbound channel adapter definition with a valid combination of attributes along with
+	 * request handler chain.
+	 *
+	 * Need AWS Connectivity.
```

Comment:
Not sure why we need `AWS Connectivity` if we should just test that `request handler chain` works properly...

From other side I'd like do not have any *real* test in this project. Just because we build here just only *adapters* around AWS services and all hard work is done there we must only `verify` that we invoke correct AWS API.

Yes, there might be some tiny cases, which should be addressed in our adapters, when we don't have a high-level API from the AWS. But we will consider them separately.

Here we just deal with the `request handler chain`, so it is really bad to try to connect to the AWS for this our trivial test...